babble/src/main/java/io/mosaicnetworks/babble/node/ConfigManager.java

```diff
@@ -115,7 +125,7 @@ private ConfigManager(Context appContext) {
 
         Log.v("ConfigManager", "got Key Pair");
 
-        File babbleDir = new File(sRootDir, BabbleConstants.BABBLE_ROOTDIR());
```

Comment:
This one is more tricky. There are a range of constants in use here - and BabbleConstants is used to allow xml files to override the defaults in other apps.